src/web/src/components/SearchResults.tsx

```diff
@@ -56,7 +56,7 @@ type SearchResultProps = {
 const SearchResults = ({ text, filter }: SearchResultProps) => {
   const classes = useStyles();
   const prepareUrl = (index: number) =>
-    `${telescopeUrl}/query?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;
+    `${postsServiceUrl}/query?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;
```

Comment:
I don't understand this change. The `Post` MS doesn'thave a `/query` endpoint, how is this going to work?

Reply:
This was mentioned in that PR. I used the wrong url there, this is supposed to be the search service being used here instead. This change shouldn't have been picked up